build.gradle

```diff
@@ -23,8 +23,15 @@ File getEnvironmentScript()
 
 apply from: environmentScript
 
+ext.publishToMaven = project.hasProperty('publishToMaven')
+if (ext.publishToMaven) {
+    plugins.apply('maven')
+    //workaround for a bug in gradle's "maven" plugin. See https://discuss.gradle.org/t/error-in-parallel-build/7215/3
```

Comment:
Better to use a consistent style for comment: one empty space after `//` and first letter capital case.